users/faq-parts/troubleshooting.rst

```diff
@@ -128,7 +128,7 @@ The web GUI contains a ``Recent Changes`` button under the device list which
 displays changes since the last (re)start of Syncthing. With the ``--audit``
 option you can enable a persistent, detailed log of changes and most
 activities, which contains a ``JSON`` formatted  sequence of events in the
```

Comment:
Nitpick: Double space.  Which is fine after a full-stop (end of sentence), but not within.

```suggestion
activities, which contains a ``JSON`` formatted sequence of events in the
```